src/scss/components/ec-card/_ec-card.scss

```diff
@@ -6,6 +6,17 @@
   background: $white;
   overflow: hidden;
 
+  &--is-interactive {
+    cursor: pointer;
+    transition: transform ease 0.4s, box-shadow ease 0.4s;
```

Comment:
Maybe this can be a mixin, @mcibique ?

Reply:
I'm not sure. I haven't seen any other component which could reuse it, but it could be extracted into something like this:
```
@mixin interactive-transition {
  @include box-shadow-level-1;

  cursor: pointer;
  transition: transform ease 0.4s, box-shadow ease 0.4s;

  &:hover {
    @include box-shadow-level-2;

    transform: translateY(-8px);
  }
}
```

I'd rather focus on performance of this thing, because transitioning box-shadow is always super slow everywhere, check this out: https://tobiasahlin.com/blog/how-to-animate-box-shadow/.

Reply:
@jaimeperezpedra @mcibique I don't think that we can reuse it, it's too specific, so this time I'd prefer not to do a mixin.

Reply:
I applied the solution you shared, many thanks.